docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/webpack.entries.js

```diff
@@ -47,4 +47,5 @@ module.exports = {
 		'src/entrypoints/app-module/app-module.ts'
 	),
 	'addl-lib-test': path.resolve(__dirname, 'src/entrypoints/addl-lib-test/addl-lib-test.ts'),
+	'addl-types-landing-page': path.resolve(__dirname, 'src/entrypoints/addl-types-landing-page/addl-types-landing-page.ts'),
```

Comment:
Put a comment in as to what this is for. The name actually confused me for a moment. Is this "Additional Types" for "Landing Page content type?". This probably should be `addl-cancer-types-landing-page` or something to make it REAL clear what it is for.